### ISSUE TITLE
wrong librosa output format?

### ISSUE THREAD

**Issue by dankorg** [andabi/music-source-separation#4]
If the input files for evaluation phase are 16bit, 16kHz, why is the output written as 32bit,16kHz?

Is there a way to automatically save the output using the same format as the input?

**Comment by andabi** [comment]
@dankorg I don't get it. what does 16bit or 32bit mean?

**Comment by dankorg** [comment]
Sorry, let me try to maybe rephrase. We are dealing with WAV files correct? They have their properties, for example, mono or stereo, bit depth, sampling frequency. In this case, the training phase of the algorithm imports the stereo WAV files encoded as signed 16bit PCM 16kHz quality. This is all fine, even the testing phase can work with either mono or stereo and any bit depth and sampling frequency. 

My question is, for the testing phase portion, if my testing WAV file is encoded as 16bit and 16kHz quality, why is the output from librosa 32bit 16kHz? Should be 16bit, since this is what the input was, but instead somewhere in the code, a conversion is done, for no reason. Output should be the same as input. I hope this helps you understand.

**Comment by andabi** [comment]
@dankorg oh, I understood. I updated the code and then it will write wav file with 16bit/16kHz ;) Thank you for the issue.

**Comment by dankorg** [comment]
Thank you for update, you are very fast! :)
